cairo/src/rectangle_int.rs

```diff
@@ -10,10 +10,45 @@ use std::mem;
 #[repr(C)]
 #[doc(alias = "cairo_rectangle_int_t")]
 pub struct RectangleInt {
-    pub x: i32,
-    pub y: i32,
-    pub width: i32,
-    pub height: i32,
+    x: i32,
+    y: i32,
+    width: i32,
+    height: i32,
+}
+
+impl RectangleInt {
+    pub fn new(self, x: i32, y: i32, width: i32, height: i32) -> Self {
+        Self {
+            x,
+            y,
+            width,
+            height,
+        }
+    }
+    pub fn x(self) -> i32 {
```

Comment:
```suggestion
    pub fn x(&self) -> i32 {
```

Reply:
Same question: should we remove the `Copy` impl then?